test/helpers/helpers.go

```diff
@@ -334,3 +348,104 @@ func CheckCscSuccessfulDeletion(client test.FrameworkClient, cscName, namespace,
 
 	return nil
 }
+
+// RestartMarketplace scales the marketplace deployment down to zero and then scales
+// it back up to it's original number of replicas, and waits for a successful deployment.
+func RestartMarketplace(client test.FrameworkClient, namespace string) error {
+	marketplace := &apps.Deployment{}
+	err := client.Get(context.TODO(), types.NamespacedName{Name: "marketplace-operator", Namespace: namespace}, marketplace)
+	if err != nil {
+		return err
+	}
+	replicas := marketplace.Spec.Replicas
+	zero := int32(0)
+	marketplace.Spec.Replicas = &zero
+	err = client.Update(context.TODO(), marketplace)
+	if err != nil {
+		return err
+	}
+	err = client.Get(context.TODO(), types.NamespacedName{Name: "marketplace-operator", Namespace: namespace}, marketplace)
+	if err != nil {
+		return err
+	}
+	marketplace.Spec.Replicas = replicas
+	err = client.Update(context.TODO(), marketplace)
+	if err != nil {
+		return err
+	}
+	err = WaitForSuccessfulDeployment(client, *marketplace)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// CreateDatastoreCsc returns a newly built CatalogSourceConfig
+func CreateDatastoreCsc(name, namespace string) *v2.CatalogSourceConfig {
```

Comment:
Please be consistent and use `CreateDatastoreCscDefinition` like we are doing with `CreateOperatorSourceDefinition`. Applies to all places.